buildpacks/ruby/tests/integration_test.rs

```diff
@@ -57,9 +62,26 @@ fn test_default_app_ubuntu20() {
 
 #[test]
 #[ignore = "integration test"]
-fn test_default_app_latest_distro() {
+fn test_default_app_ubuntu22() {
     TestRunner::default().build(
         BuildConfig::new("heroku/builder:22", "tests/fixtures/default_ruby"),
+        |context| {
+            println!("{}", context.pack_stdout);
+            assert_contains!(context.pack_stdout, "# Heroku Ruby Buildpack");
+            assert_contains!(
+                context.pack_stdout,
+                r#"`BUNDLE_BIN="/layers/heroku_ruby/gems/bin" BUNDLE_CLEAN="1" BUNDLE_DEPLOYMENT="1" BUNDLE_GEMFILE="/workspace/Gemfile" BUNDLE_PATH="/layers/heroku_ruby/gems" BUNDLE_WITHOUT="development:test" bundle install`"#);
+
+            assert_contains!(context.pack_stdout, "Installing webrick");
+        },
+    );
+}
+
+#[test]
+#[ignore = "integration test"]
+fn test_default_app_latest_distro() {
```

Comment:
(Can be handled in a later PR) There is currently no testing of arm64 in CI. 

However, adding that would meaning needing to switch to a matrix style of testing, where the tests themselves all run on the input builder, and CI picks a different builder (and potentially architecture) per job in the matrix. For Python I make sure to run all python version related tests on all builders (so I know each compiled python archive works as expected, can find all dynamic libraries in that builder, hasn't been accidentally overwritten on S3 etc), and then on the main builder I run all tests.

As such, I'm fine with you making those CI changes in a later PR.